packages/design/src/tokens/baseColor.tokens.json

```diff
@@ -293,34 +293,98 @@
       },
       "orange-": {
         "100": {
-          "$value": "hsl(36, 71%, 97%)"
+          "$value": "hsl(31, 71%, 97%)"
         },
         "200": {
-          "$value": "hsl(34, 76%, 92%)"
+          "$value": "hsl(31, 74%, 93%)"
         },
         "300": {
-          "$value": "hsl(32, 78%, 84%)"
+          "$value": "hsl(31, 72%, 85%)"
         },
         "400": {
-          "$value": "hsl(33, 77%, 63%)"
+          "$value": "hsl(31, 83%, 64%)"
         },
         "500": {
-          "$value": "hsl(33, 71%, 48%)"
+          "$value": "hsl(31, 80%, 48%)"
         },
         "600": {
-          "$value": "hsl(33, 71%, 38%)"
+          "$value": "hsl(31, 90%, 37%)"
         },
         "700": {
-          "$value": "hsl(33, 71%, 29%)"
+          "$value": "hsl(31, 100%, 27%)"
         },
         "800": {
-          "$value": "hsl(33, 72%, 20%)"
+          "$value": "hsl(31, 100%, 18%)"
         },
         "900": {
-          "$value": "hsl(33, 72%, 13%)"
+          "$value": "hsl(31, 90%, 12%)"
```

Comment:
Standardize the hue across the orange range and drive up saturation, particularly on the darker (500–900) steps to maintain more of an "orange" as long as possible